### ISSUE TITLE
Select not visible in high contrast mode

### ISSUE THREAD

**Issue by crisbeto** [angular/material2#6383]
The select bottom border isn't rendered in high contrast mode which makes it indistinguishable from regular text. Another issue is that the arrow is rendered as a rectangle, however I'm not sure whether we can do much about it. We can try with SVG, but I believe that it won't be rendered at all. For reference:

![angular_material_-_internet_explorer_2017-08-09_22-20-59](https://user-images.githubusercontent.com/4450522/29142089-0aeeb436-7d51-11e7-883a-863980dab332.png)
